index.html

```diff
@@ -2825,125 +2826,274 @@ <h3>Working with personal names</h3>
 </aside>
 
 
+<section id="loc_names_field_length">
+<h4>Field length &amp; composition</h4>
+
 	<div class="req" id="loc_names_given_family">
-	<p class="advisement">Check whether you really need to store or access given name and family name separately.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">Check whether you really need to store or access given name(s) and family name(s) separately.</p>
```

Comment:
I would just eliminate the parentheses and say `...access given names and family names...`